File: src/js/power_float.js
const { performance } = require('perf_hooks');

const base = parseFloat(process.env.BASE) || 2.0;
const exp = parseInt(process.env.EXP) || 1023;

const start = performance.now();
const result = Math.pow(base, exp);
const end = performance.now();

console.log(`Result(${base}^${exp}): ${result.toExponential(10)}`);
console.log(`Time: ${(end - start).toFixed(3)} ms`);

File: src/python/power_float.py
import os
import time
import math

def main():
    try:
        base = float(os.environ.get("BASE", 2.0))
    except ValueError:
        base = 2.0
    
    try:
        exp = int(os.environ.get("EXP", 1023))
    except ValueError:
        exp = 1023

    start = time.perf_counter()
    result = math.pow(base, exp)
    end = time.perf_counter()
    
    print(f"Result({base}^{exp}): {result:.10e}")
    print(f"Time: {(end - start) * 1000:.3f} ms")

if __name__ == "__main__":
    main()
